srai/regionizers/voronoi_regionizer.py

```diff
@@ -0,0 +1,68 @@
+"""
+Voronoi Regionizer.
+
+This module contains voronoi regionizer implementation.
+"""
+
+from itertools import combinations
+
+import geopandas as gpd
+
+from ._spherical_voronoi import generate_voronoi_regions
+
+
+class VoronoiRegionizer:
+    """A Voronoi regionizer."""
+
+    def __init__(self, seeds: gpd.GeoDataFrame, max_meters_between_points: int = 10_000) -> None:
+        """
+        VoronoiRegionizer.
+
+        Voronoi [1] regionizer allows the given geometries to be divided into
```

Comment:
Should it be written in the `__init__` method or under the class name?

Reply:
Leaving it for now under `__init__`.